Source/VolunteerReporting/Web/Controllers/AutomaticRepliesController.cs

```diff
@@ -0,0 +1,48 @@
+using Events.External;
+using Infrastructure.AspNet;
+using Microsoft.AspNetCore.Mvc;
+using Read.AutomaticReplyMessages;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Web.Models;
+
+namespace Web.Controllers
+{
+    [Route("api/automaticreplies")]
+    public class AutomaticRepliesController : BaseController
+    {
+        private readonly IDefaultAutomaticReplies _defaultAutomaticReplies;
+
+        public AutomaticRepliesController(IDefaultAutomaticReplies defaultAutomaticReplies)
+        {
+            _defaultAutomaticReplies = defaultAutomaticReplies;
+        }
+
+        [HttpGet("automaticreplytypes")]
+        public IEnumerable<AutomaticReplyTypeModel> GetAutomaticReplyTypes()
+        {
+            return Enum.GetValues(typeof(DefaultAutomaticReplyType)).Cast<DefaultAutomaticReplyType>().Select(c => new AutomaticReplyTypeModel()
+            {
+                Id = (int)c,
+                Name = c.ToString()
+            });
+        }
+
+        [HttpGet("{projectId}")]
+        public async Task<IEnumerable<AutomaticReplyModel>> GetAutomaticRepliesForProject(Guid projectId)
+        {
+            var defaults = await _defaultAutomaticReplies.GetAllAsync();
+
+            // TODO: Get replies for project and filter any custom from the default list
+
+            return defaults.Select(c => new AutomaticReplyModel() {
+                Auto = true,
```

Comment:
Question; What does Auto mean in this context - seeing the model is called AutomaticReply.

Reply:
Sorry, this should be called default. Used for distinguishing from the project specific and the global (default) reply messages. Good catch!